src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Providers/AbstractSemanticQuickInfoProvider.cs

```diff
@@ -276,8 +276,10 @@ internal abstract partial class AbstractSemanticQuickInfoProvider : AbstractQuic
             var symbols = semanticModel.GetSymbols(token, document.Project.Solution.Workspace, bindLiteralsToUnderlyingType: true, cancellationToken: cancellationToken);
 
             var bindableParent = document.GetLanguageService<ISyntaxFactsService>().GetBindableParent(token);
-            var overloads = semanticModel.GetMemberGroup(bindableParent, cancellationToken);
-
+            var parentSymbol = semanticModel.GetSymbolInfo(bindableParent).Symbol;
+            var overloads = semanticModel.GetMemberGroup(bindableParent, cancellationToken)
+                                         .Where(s => parentSymbol == null || enclosingType.InheritsFromOrEquals(parentSymbol.ContainingType) || s.IsStatic == parentSymbol.IsStatic);
+            
```

Comment:
Not trying to make a statement about the correctness of this approach, but how similar is it to the code in completion which already handles static/instance?